region/wreckedship/main.json

```diff
@@ -457,24 +457,24 @@
                 {
                   "name": "Shoot the Blocks",
                   "notable": false,
-                  "requires": [
-                    {"or": [
-                      "canLateralMidAirMorph",
-                      "h_canUseSpringBall"
-                    ]}
-                  ],
+                  "requires": [ "Morph" ],
+                  "note": "Simple jump morph from the top of the stairs.",
```

Comment:
I think I would refrain from calling it 'simple', as it's fairly tricky compared to basic mid-air-morphs. Maybe something like:
"Jump and morph from the top of the stairs. This is easier with Hi-Jump Boots and Speed Booster turned off."?

Reply:
I can see that with slower button presses it can be harder than I make it sound.  I would suggest crouch jump on the stairs for an easier setup, but maybe people know the run jump better or would just expect to try it that way.